<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textInput"/>
    </item>
    <item>
     <widget class="QPushButton" name="copyTextButton">
      <property name="text">
       <string>Click me!</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stickyButton">
      <property name="text">
       <string>Don't click me!</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textOutput">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
